content/docs/5.1/deploy/install/prerequisites.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Prerequisites
+description: Learn the software and hardware recommendations for deploying and running TiKV
+menu:
+    "5.1":
+        parent: Install TiKV
+        weight: 1
+---
+As an open source distributed key-value database, TiKV can be deployed in the Intel architecture server, ARM architecture server, and major virtualization environments and runs well. TiKV supports most of the major hardware networks and Linux operating systems.
```

Comment:
I can't see the connection between `open source distributed key-value database` and cross-platform. May be better to split this sentence into two statements.

Reply:
sure